html-aam/roles-contextual.html

```diff
@@ -25,6 +25,7 @@
 <nav>
   <aside data-testname="el-aside-in-section-with-name" data-expectedrole="complementary" aria-label="x" class="ex">x</aside>
   <aside data-testname="el-aside-in-section-without-name" class="ex-generic">x</aside>
+  <aside data-testname="el-aside-in-section-with-role" data-expectedrole="complementary" class="ex" role="complementary">x</aside>
```

Comment:
```suggestion
  <aside data-testname="el-aside-in-section-without-name-and-with-complementary-role" data-expectedrole="complementary" class="ex" role="complementary">x</aside>
```

Reply:
I think we can probably accept this as-is as a downstream export, and update the test name in the later PR: 
- #43013